op-devstack/sysgo/l2_cl_opnode.go

```diff
@@ -232,6 +267,15 @@ func WithOpNode(l2CLID stack.L2CLNodeID, l1CLID stack.L1CLNodeID, l1ELID stack.L
 			}
 		}
 
+		// Determine the L2 endpoint based on whether we're using a SyncTester or L2EL
+		var l2EngineAddr string
+		if useSyncTester {
+			require.NotNil(orch.syncTester, "sync tester service required when using SyncTester")
+			l2EngineAddr = orch.syncTester.service.SyncTesterEndpoint(syncTesterID.ChainID(), tb)
```

Comment:
My Idea is that, we should leave the orch.syncTester.service.SyncTesterEndpoint as is, and add the suffix addition to the url part here:
```
	endpoint := fmt.Sprintf("%s/chain/%s/synctest/%s", s.RPC(), chainID, uuid)
	if target != nil {
		endpoint += fmt.Sprintf("?head=%d&safe=%d&finalized=%d", target.Head, target.Safe, target.Finalized)
	}
```
Since at this point we know the `tb`, and no need to embed this into the config.